macros/plugins/snowflake/snowpipe/get_copy_sql.sql

```diff
@@ -25,13 +25,14 @@
         {% endif %}
             metadata$filename::varchar as metadata_filename,
             metadata$file_row_number::bigint as metadata_file_row_number,
+            metadata$file_last_modified::timestamp as metadata_file_last_modified,
             current_timestamp::timestamp as _dbt_copied_at
```

Comment:
@jtmcn [this doc](https://docs.snowflake.com/en/user-guide/querying-metadata) you shared mentioned `https://docs.snowflake.com/en/user-guide/querying-metadata`?
👀 @Catisyf 
```suggestion
            metadata$start_scan_time as _dbt_copied_at
```